### PR TITLE
Add Asset Store chart

### PR DESCRIPTION
**Pull request by michal-hudy**
**Description**

Changes proposed in this pull request:

- Added Asset Store chart
- Moved Minio chart to Asset Store
- Updated Minio

**Related issue(s)**

#2459 
#2409

### CODE REVIEW

File: resources/assetstore/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+3. Click **Create service account**, name your account, and click **Create**.
+4. Set the `Storage Admin` role.
+5. Click **Create key** and choose `JSON` as a key type.
+6. Save the `JSON` file.
+7. Create a Secret from the JSON file by running this command:
+    ```bahs
+    kubectl create secret generic assetstore-gcs-credentials --from-file=service-account.json={filename} --namespace kyma-system
+    ```
+
+### Update Asset Store deployment
```
Comment (kazydek) [inline resources/assetstore/README.md]:
```suggestion
### Update the Asset Store deployment
```

File: installation/resources/installer-config-local.yaml.tpl [hunk 2]
```diff
@@ -103,3 +91,15 @@ metadata:
...
+kind: ConfigMap
+metadata:
+  name: assetstore-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: assetstore
+data:
+  minio.resources.requests.memory: 64Mi
+  minio.resources.limits.cpu: 100m
```
Comment (sjanota) [inline installation/resources/installer-config-local.yaml.tpl]: Shouldn't it be set in `values.yaml`?

File: resources/assetstore/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+To update Asset Store deployment run this command:
+
+```bash
+helm upgrade assetstore resources/assetstore --namespace kyma-system --wait=true --reuse-values --set minio.persistence.enabled=false --set minio.gcsgateway.enabled=true --set minio.gcsgateway.replicas=1 --set minio.gcsgateway.gcsKeySecret=assetstore-gcs-credentials --set minio.gcsgateway.projectId={gcp-project} --set minio.defaultBucket.enabled=false
+```
+
+>**NOTE:** This is alpha version of Asset Store and after migration to GCS `content` bucket will be not available. Also buckets in GCS must by globally unique.
```
Comment (kazydek) [inline resources/assetstore/README.md]:
```suggestion
>**NOTE:** This is an alpha version of the Asset Store. In this version, the GCS `content` bucket is not available in the Minio Gateway mode. There is also no verification that checks if the bucket name is unique, which is a requirement in GCS.

File: installation/resources/installer-config-local.yaml.tpl (hunk 2 above)
Comment (michal-hudy) [reply installation/resources/installer-config-local.yaml.tpl]: This is an override for local development - minikube. In `values.yaml` we have a production setup.

File: resources/assetstore/charts/assetstore-controller-manager/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Asset Store Controller Manager
+
+## Overview
+
+The Asset Store Controller Manager manages creation of buckets and uploading assets to storage server like Google Cloud Storage.
```
Comment (kazydek) [inline resources/assetstore/charts/assetstore-controller-manager/README.md]:
```suggestion
The Asset Store Controller Manager manages the creation of buckets and upload of assets to such storage servers as the Google Cloud Storage.
```

File: resources/assetstore/charts/assetstore-controller-manager/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+## Details
+
+For more information, see the README document in the [Asset Store Controller Manager](https://github.com/kyma-project/kyma/tree/master/components/assetstore-controller-manager) repository.
```
Comment (kazydek) [inline resources/assetstore/charts/assetstore-controller-manager/README.md]:
```suggestion
For more information, see the `README.md` document in the [Asset Store Controller Manager](https://github.com/kyma-project/kyma/tree/master/components/assetstore-controller-manager) repository.
```

**Pull request merged**